### PR TITLE
Sprint2 work schedule screen

### LINKED ISSUE #36: Work Schedule Functionality

### PR DESCRIPTION
**Pull request by vrajp13**
- Added a work schedule component to the home screen
- Merged with main as well to make sure no merge conflicts occur
- Also added another dependency of materials-ui to allow for more UI components and icons

### CODE REVIEW

File: components/work-schedule/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+const WorkSchedule: React.FunctionComponent<WorkScheduleProp> = (props) => {
+  return (
+    <div className={styles.div}>
+      <List className={styles.list}>
+        <ListItem className={styles.listItem}>
+          <ListItemAvatar>
+            <Avatar>
+              <AlarmRoundedIcon style={{ color: red[500] }} />
```
Comment (ibaniya98) [inline components/work-schedule/index.tsx]: Do you want to make this white? I think red is too contrasting with all the colors that we have.

File: components/work-schedule/work-schedule.module.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.div {
+  margin-top: 1.75rem;
+  margin-bottom: 1.75rem;
```
Comment (ibaniya98) [inline components/work-schedule/work-schedule.module.css:3]: Add `border-radius: 10px`. Helps to have a rounded shape to be consistent with existing designs.

File: pages/home.tsx [hunk 3]
```diff
@@ -47,6 +48,12 @@ export function Home(props: HomeProps): JSX.Element {
           <Button variant="warning" href="/api/auth/signout" className="mt-5">
             Sign out
           </Button>
+
+          <WorkSchedule
+            workDay="02/26/2021"
+            startTime="11:30:00"
```
Comment (ibaniya98) [inline pages/home.tsx]: I say hide the seconds for now. No reason to display it.

**Review by ibaniya98 — DISMISSED** [review]
Looks good, but if you want, you can address some minor issues I have suggested above.

**Comment by vrajp13** [comment]
Made changes according to the suggestions.

**Review by gsuStudent — DISMISSED** [review]

**Comment by vrajp13** [comment]
- Minor change to work schedule icons
- Minor changes to the Registration screen to align the labels

**Comment by cpc2820** [comment]
Looks good to me. The text looks aligned now on the registration screen.

**Review by cpc2820 — APPROVED** [review]

**Pull request merged**